internal/adapters/uidriver/server.go

```
@@ -0,0 +1,394 @@
+		s.mu.Unlock()
+		_ = stream.Close()
+		return &ports.UIError{Code: ports.UIErrBusy}
+	}
+	s.connections++
+	s.mu.Unlock()
+	defer func() {
+		s.mu.Lock()
+		s.connections--
+		s.mu.Unlock()
+	}()
+	defer stream.Close()
```

golangci-lint (2.13.2)

[error] Error return value of `stream.Close` is not checked

(errcheck)